skunkworks_crow/src/main/java/org/odk/share/tasks/UploadJob.java

```diff
@@ -513,6 +521,12 @@ private void sendInstances(List<String> instanceIds, int progress, int total) {
         }
     }
 
+    private void setupResultTest(String formName, String status) {
+        sbResult.append(getContext().getString(R.string.form_name, formName) + "\n");
+        sbResult.append(getContext().getString(R.string.form_transfer_result, ", " + status));
```

Comment:
I think `,` should not be there as it will result into something like this
`Result: ,Success Sent for review`

Reply:
For string format, this case won't appears. `,` is always behind `Success`

Reply:
`R.string.form_transfer_result` is something like ` Success %s`.

Reply:
Nope, it's not the case can you check this once by transferring filled form.

Reply:
`R.string.form_transfer_result` is like `Result: %s` and `%s` is ` "," + status`

Reply:
Ohh, you are right, let me fix that.

Reply:
updated, please review again